coursedata/adventures/de.yaml

```
@@ -403,32 +403,38 @@ adventures:
                     ```
                 start_code: ""
     songs:
-        name: "Sing a song!"
-        description: "Print a song"
+        name: "Singe ein Lied!"
+        description: "Geben einen Songtext aus."
         image: "songs.png"
         default_save_name: "Song"
         levels:
             12:
                 story_text: |
-                    ## Sing a song!
-                    In this song we can make it even easier to program 'if you're happy and you know it, clap your hands'. Because we can put all of the actions in a variable, check it out:
-                    ## Example Hedy code
+                    ## Lieder
+                    Jetzt machen wir es und richtig einfach und programmieren 'Wenn Du glücklich bist, dann klatsche in die Hand'.
+                    Alles, was wir tun möchten, wenn wir glücklich sind, speichern wir in einer Variablen.
```

This is a rather free translation of
> In this song we can make it even easier to program 'if you're happy and you know it, clap your hands'. Because we can put all of the actions in a variable […]

If we want to capture the meaning of the English version somewhat more closely, we could maybe do that like so:
```suggestion
                    In diesem Lied können wir 'Wenn Du glücklich bist, dann klatsche in die Hand' sogar noch einfacher programmieren.
                    Denn alles, was wir tun möchten, wenn wir glücklich sind, können wir in einer Variablen speichern.
```

---

Absolutely. I opted for this translation because it's the first occurence of this song in this adventure, so the original English text doesn't make sense. I figured that the adventure has probably been changed somewhere in the past, so the original text is a relict and I changed it accordingly. You think it's fine?